### PR TITLE
`1.into()` -> `ParaId::from(1_u32)`

### PR DESCRIPTION
**Pull request by drahnr**
Be explicit about the types. `rustc` causes for this particular one a myriad of error messages when an unrelated issue pops up.

Extracted from #5314

### CODE REVIEW

File: runtime/parachains/src/paras_inherent/mod.rs [hunk 1]
```diff
@@ -535,7 +535,8 @@ impl<T: Config> Pallet<T> {
 
 		// Give some time slice to dispatch pending upward messages.
 		// this is max config.ump_service_total_weight
-		let _ump_weight = <ump::Pallet<T>>::process_pending_upward_messages();
+		let total_consumed_weight =
+			total_consumed_weight + <ump::Pallet<T>>::process_pending_upward_messages();
```
Comment (ordian) [inline runtime/parachains/src/paras_inherent/mod.rs:539]: this change is unrelated to the PR?

Comment (drahnr) [reply runtime/parachains/src/paras_inherent/mod.rs:539]: Good catch.

**Review by ordian — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

**Comment by drahnr** [comment]
bot merge

**Comment by drahnr** [comment]
bot merge

**Comment by drahnr** [comment]
bot merge
